### DISCUSSION TITLE
✨ Stars ✨

### DISCUSSION THREAD

**Discussion by Dominik48N** [Schweinepriester/github-profile-achievements#362] [General]
https://github.com/Dominik48N/jda-commands

**Comment by scarabaeus** [comment]
@Dominik48N Done! https://github.com/scarabaeus/scarabaeus

**Reply by WHAHA-HA** [reply]
@scarabaeus, done. https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Comment by WHAHA-HA** [comment]
@Dominik48N, done.
https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Comment by Ashu-Barnwal** [comment]
DONE
https://github.com/Ashu-Barnwal/Color-Plate

**Comment by bacca87** [comment]
@Dominik48N DONE
https://github.com/bacca87/great2

**Comment by polekstulod** [comment]
Done. https://github.com/polekstulod/foodtrip-api

**Comment by ghostinthematrix0** [comment]
https://github.com/mendeeislife/mendeeislife
